src/main/java/com/android/ex/chips/RecipientEditTextView.java

```diff
@@ -3163,6 +3163,16 @@ public BaseRecipientAdapter getAdapter() {
      * unfinished text at the end.
      */
     public void appendRecipientEntry(final RecipientEntry entry) {
+        if(getMeasuredWidth() == 0){
+            post(new Runnable() {
+                @Override
+                public void run() {
+                    appendRecipientEntry(entry);
+                }
+            });
+            return;
+        }
```

Comment:
it's not related to identity id support, it's a minor fix we added, in order to make sure we have width of chip when adding it to the edittext. (we encountered a scenario where we might not have width when adding a chip)' because if we don't have width, the chip is inflated with a default tiny width.